packages/core/tests/types.test.ts

```diff
@@ -84,18 +84,49 @@ const PotatoButton = css.css({
 			},
 		},
 	},
+	compoundVariants: [
+		{
+			variant: 'blue',
+			css: {
+				backgroundColor: '$gray200',
+			},
+		},
+	],
+})
+
+const two = css.css(PotatoButton, {
+	$$max: '2px',
+	width: '$$max',
+	variants: {
+		variant: {
+			green: {
+				width: '$$max',
```

Comment:
May we add tests for overriding `$$max` within here? Similarly within any place `when` is supported?